stubs/docker/docker/api/exec_api.pyi

```diff
@@ -9,12 +17,69 @@ class ExecApiMixin:
         tty: bool = False,
         privileged: bool = False,
         user: str = "",
-        environment=None,
-        workdir=None,
-        detach_keys=None,
-    ): ...
-    def exec_inspect(self, exec_id): ...
-    def exec_resize(self, exec_id, height=None, width=None) -> None: ...
+        environment: dict[str, str] | list[str] | None = None,
+        workdir: str | None = None,
+        detach_keys: str | None = None,
+    ) -> dict[str, Incomplete]: ...
+    def exec_inspect(self, exec_id: str) -> dict[str, Incomplete]: ...
+    def exec_resize(self, exec_id: str, height: int | None = None, width: int | None = None) -> None: ...
+    @overload
     def exec_start(
-        self, exec_id, detach: bool = False, tty: bool = False, stream: bool = False, socket: bool = False, demux: bool = False
-    ): ...
+        self,
+        exec_id: str,
+        detach: Literal[True] = ...,
```

Comment:
This wouldn't work, since it would also match if `detach` is left out (meaning it's `False`):


```suggestion
        detach: Literal[True],
```